collector/accelerators_test.go

```diff
@@ -44,7 +45,9 @@ func TestAccelerator(t *testing.T) {
 	node_accelerator_card_info{id="0000:00:1f.5",model="RTX_4090",vendor="NVIDIA"} 1
 	`
 
-	*sysPath = "fixtures/sys"
+	if _, err := kingpin.CommandLine.Parse([]string{"--collector.accelerators.configmap_path", "testdata/accelerators_test_data.yaml", "--path.sysfs", "fixtures/sys"}); err != nil {
```

Comment:
I think this is better done in a new test where we call `prepareVendorModelData` directly instead of simulating a kingpin parsing.
(CMO e2e tests ensure that parsing is done correctly in the prod code)

Reply:
fixed